core/shared/src/main/scala/scalaz/zio/IO.scala

```diff
@@ -0,0 +1,571 @@
+package scalaz.zio
+
+import scalaz.zio.Exit.Cause
+import scalaz.zio.internal.{ Executor, Platform }
+
+import scala.concurrent.ExecutionContext
+
+/*
+ * PHASE 1 DONE
+ * PHASE 2 DONE
+ * PHASE 3 DONE
+ * PHASE 4 DONE
+ * PHASE 6 DONE
+ */
+
+/*
+ * EXTENDS ZIO_E_ANY
+ */
+object IO {
+
+  def apply[A](a: => A): Task[A] = effect(a)
+
+  /*
+   * ------------------------------------------------------------------------------------------------
+   * The Following is taken from ZIOFunctions.
+   * ------------------------------------------------------------------------------------------------
+   */
+  // ALL error types in this trait must be a subtype of `UpperE`.
+  // TAKEN FROM ZIO_E_THROWABLE
+  // ALL environment types in this trait must be a supertype of `LowerR`.
+  type IoR = Any
+
+  // type IoE = Polymorphic
+
+  /** Returns an effect that models failure with the specified error.
+   * The moral equivalent of `throw` for pure code.
+   */
+  final def fail[E](error: E): IO[E, Nothing] = ZIO.fail(error)
+
+  /** Returns an effect that models failure with the specified `Cause`.
+   */
+  final def halt[E](cause: Cause[E]): IO[E, Nothing] = ZIO.halt(cause)
+
+  /** Returns an effect that models success with the specified strictly-
+   * evaluated value.
+   */
+  final def succeed[A](a: A): UIO[A] = ZIO.succeed(a)
+
+  /** Returns an effect that models success with the specified lazily-evaluated
+   * value. This method should not be used to capture effects. See
+   * `[[ZIO.effectTotal]]` for capturing total effects, and `[[ZIO.effect]]` for capturing
+   * partial effects.
+   */
+  final def succeedLazy[A](a: => A): UIO[A] = ZIO.succeedLazy(a)
+
+  /** Accesses the whole environment of the effect.
+   */
+  final def environment: ZIO[IoR, Nothing, IoR] = ZIO.access(ZIO.identityFn[IoR])
+
+  /** Accesses the environment of the effect.
+   * {{{
+   * val portNumber = effect.access(_.config.portNumber)
+   * }}}
+   */
+  final def access: ZIO.AccessPartiallyApplied[IoR] =
+    ZIO.access
+
+  /** Effectfully accesses the environment of the effect.
+   */
+  final def accessM: ZIO.AccessMPartiallyApplied[IoR] =
```

Comment:
Ditto for this, we know `R = Any`, so we can delete this method, since it won't be useful for `IO`.